<!DOCTYPE html> <html lang="en"> <head> <meta name="google-site-verification" content="RdkFS1JqlMGtjHL3GHcfRzAuJ1JVMGlT_KWyjGv5p9w"/> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Bryan Merrill</title> <meta name="author" content="Bryan Merrill"/> <meta name="description" content="Personal website for Bryan Merrill "/> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"/> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"/> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="none" id="highlight_theme_light"/> <link rel="shortcut icon" href="/assets/img/phage_icon.png"/> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://brymerr921.github.io/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"/> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> <script async src="https://www.googletagmanager.com/gtag/js?id="></script> <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","");</script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-0551J88NK3"></script> <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-0551J88NK3");</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <div class="navbar-brand social"> <a href="mailto:%62%72%79%6D%65%72%72%39%32%31@%67%6D%61%69%6C.%63%6F%6D" title="email"><i class="fas fa-envelope"></i></a> <a href="https://orcid.org/0000-0002-9967-5539" title="ORCID" target="_blank" rel="noopener noreferrer"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=C__eM6YAAAAJ" title="Google Scholar" target="_blank" rel="noopener noreferrer"><i class="ai ai-google-scholar"></i></a> <a href="https://www.researchgate.net/profile/https://www.researchgate.net/profile/Bryan-Merrill/" title="ResearchGate" target="_blank" rel="noopener noreferrer"><i class="ai ai-researchgate"></i></a> <a href="https://github.com/brymerr921" title="GitHub" target="_blank" rel="noopener noreferrer"><i class="fab fa-github"></i></a> <a href="https://www.linkedin.com/in/bryan-merrill" title="LinkedIn" target="_blank" rel="noopener noreferrer"><i class="fab fa-linkedin"></i></a> <a href="https://twitter.com/bryan_d_merrill" title="Twitter" target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter" style="color:#55acee;"></i></a> </div> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">curriculum vitae</a> </li> <div class="toggle-container"> <a id="light-toggle"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </a> </div> </ul> </div> </div> </nav> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Bryan</span> Merrill </h1> <p class="desc"></p> </header> <article> <div class="profile float-right"> <figure> <picture> <source media="(max-width: 480px)" srcset="/assets/img/bryan-480.webp"></source> <source media="(max-width: 800px)" srcset="/assets/img/bryan-800.webp"></source> <source media="(max-width: 1400px)" srcset="/assets/img/bryan-1400.webp"></source> <img class="img-fluid z-dept-1 rounded" src="/assets/img/bryan.jpg" alt="bryan.jpg"> </picture> </figure> </div> <div class="clearfix"> <p>Hi, I’m Bryan! I’m a Bioinformatics Scientist with a passion for discovering new biology using multi-omics datasets.</p> <p>My work has focused primarily on (1) unlocking host-phenotype relationships using multi-omics data, (2) discovering new microbes and natural products that are relevant for health and disease, and (3) learning whether “but it feeds your good bacteria!” is sufficient motivation to get my kids to eat their broccoli.</p> <p>I’m known for contributing to a positive team environment and building successful collaborations between bench and computational researchers.</p> <h2 id="experience">experience</h2> <h5 id="nuanced-health-2022-2023"> <a href="https://www.nuancedhealth.com" target="_blank" rel="noopener noreferrer">Nuanced Health</a> (2022-2023)</h5> <p>I worked as a Bioinformatics Scientist with an incredible team, where we harnessed robust phenotypic and multi-omics data to discover new ways to augment human health and treat disease. We (1) developed and optimized many software pipelines to maximize efficiency and data quality, tailor-fit to the experimental design, and (2) leveraged these data to identify features associated with disease and drug-response states.</p> <h5 id="stanford-university-2015-2022"> <a href="https://www.stanford.edu" target="_blank" rel="noopener noreferrer">Stanford University</a> (2015-2022)</h5> <p>I completed my Ph.D. in the <a href="http://sonnenburglab.stanford.edu/" target="_blank" rel="noopener noreferrer">Sonnenburg Lab</a>, <a href="https://microimmuno.stanford.edu/" target="_blank" rel="noopener noreferrer">Department of Microbiology and Immunology</a> at <a href="https://www.stanford.edu" target="_blank" rel="noopener noreferrer">Stanford University</a> <a href="https://med.stanford.edu/" target="_blank" rel="noopener noreferrer">School of Medicine</a>. I worked on the discovery of lifestyle-associated microbes, pathogens, and bacteriophages through deep metagenome sequencing of the gut microbiome.</p> <h5 id="brigham-young-university-2011-2015"> <a href="https://www.byu.edu" target="_blank" rel="noopener noreferrer">Brigham Young University</a> (2011-2015)</h5> <p>I completed my M.S. and B.S. in the Hope Lab, <a href="https://mmbio.byu.edu/" target="_blank" rel="noopener noreferrer">Department of Microbiology and Molecular Biology</a>. I worked on phage therapy for agricultural pathogens, from discovery and characterization of novel phages through treatment.</p> <h2 id="research-interests">research interests</h2> <p>Bioinformatics, human gut microbiome, microbial genomics, phage therapy.</p> </div> <div class="news"> <h2>news</h2> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row">Aug 25, 2023</th> <td> After an amazing experience at <a href="https://www.nuancedhealth.com/" target="_blank" rel="noopener noreferrer">Nuanced Health</a> I’m now looking for my next opportunity! Please feel free to <a href="mailto:%62%72%79%6D%65%72%72%39%32%31@%67%6D%61%69%6C.%63%6F%6D" title="email">reach out</a> if you think I can help. </td> </tr> <tr> <th scope="row">Jun 21, 2023</th> <td> Exciting paper out today in Cell: <a href="https://www.cell.com/cell/fulltext/S0092-8674(23)00597-4" target="_blank" rel="noopener noreferrer">Ultra-deep sequencing of Hadza hunter-gatherers recovers vanishing gut microbes</a>. Big thanks to co-first authors Matt Carter and Matt Olm for all your help! </td> </tr> <tr> <th scope="row">Sep 22, 2022</th> <td> Excited to join the amazing team at <a href="https://www.nuancedhealth.com/" target="_blank" rel="noopener noreferrer">Nuanced Health</a> as a Bioinformatics Scientist! Looking forward to using multi-omics data to understand microbiome-host-disease interactions. </td> </tr> <tr> <th scope="row">Jul 29, 2022</th> <td> I successfully defended my PhD thesis! <img class="emoji" title=":sparkles:" alt=":sparkles:" src="https://github.githubassets.com/images/icons/emoji/unicode/2728.png" height="20" width="20"> Huge thanks to my amazing committee: Justin Sonnenburg, Ami Bhatt, Michael Fischbach, KC Huang, David Schneider, and Robert Daines (chair). </td> </tr> </table> </div> </div> <div class="publications"> <h2>selected publications</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Cell</abbr></div> <div id="RN2510" class="col-sm-8"> <div class="title">Ultra-deep sequencing of Hadza hunter-gatherers recovers vanishing gut microbes</div> <div class="author"> <a href="http://mattcarter.co/" target="_blank" rel="noopener noreferrer">Carter, Matthew M.</a>, <a href="https://mrolm.github.io/" target="_blank" rel="noopener noreferrer">Olm, Matthew R.</a>,  <em>Merrill, Bryan D.</em>, Dahan, Dylan, Tripathi, Surya, Spencer, Sean P., <a href="http://brianyu.org/" target="_blank" rel="noopener noreferrer">Yu, Feiqiao B.</a>, <a href="https://www.sunitjain.com/" target="_blank" rel="noopener noreferrer">Jain, Sunit</a>, Neff, Norma, <a href="https://nyuad.nyu.edu/en/academics/divisions/science/faculty/aashish-jha.html" target="_blank" rel="noopener noreferrer">Jha, Aashish R.</a>, <a href="http://sonnenburglab.stanford.edu/" target="_blank" rel="noopener noreferrer">Sonnenburg, Erica D.</a>, and <a href="http://sonnenburglab.stanford.edu/" target="_blank" rel="noopener noreferrer">Sonnenburg, Justin L.</a> </div> <div class="periodical"> <em>Cell</em> 2023 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://biorxiv.org/content/10.1101/2022.03.30.486478v2" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">bioRxiv</a> <a href="https://pubmed.ncbi.nlm.nih.gov/37348505" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PubMed</a> <a href="https://doi.org/10.1016/j.cell.2023.05.046" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a> <a href="https://www.cell.com/action/showPdf?pii=S0092-8674%2823%2900597-4" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a> </div> <div class="abstract hidden"> <p>The gut microbiome modulates immune and metabolic health. Human microbiome data are biased toward industrialized populations, limiting our understanding of non-industrialized microbiomes. Here, we performed ultra-deep metagenomic sequencing on 351 fecal samples from the Hadza hunter-gatherers of Tanzania and comparative populations in Nepal and California. We recovered 91,662 genomes of bacteria, archaea, bacteriophages, and eukaryotes, 44% of which are absent from existing unified datasets. We identified 124 gut-resident species vanishing in industrialized populations and highlighted distinct aspects of the Hadza gut microbiome related to in situ replication rates, signatures of selection, and strain sharing. Industrialized gut microbes were found to be enriched in genes associated with oxidative stress, possibly a result of microbiome adaptation to inflammatory processes. This unparalleled view of the Hadza gut microbiome provides a valuable resource, expands our understanding of microbes capable of colonizing the human gut, and clarifies the extensive perturbation induced by the industrialized lifestyle.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Nature</abbr></div> <div id="RN2169" class="col-sm-8"> <div class="title">A metabolomics pipeline for the mechanistic interrogation of the gut microbiome</div> <div class="author"> <a href="https://thehanlab.com/" target="_blank" rel="noopener noreferrer">Han, Shuo</a>, Van Treuren, Will, Fischer, Curt R.,  <em>Merrill, Bryan D.</em>, DeFelice, Brian C., Sanchez, Juan M., Higginbottom, Steven K., <a href="https://ltcguthrie.github.io/" target="_blank" rel="noopener noreferrer">Guthrie, Leah</a>, Fall, Lalla A., <a href="https://www.doddlab.org/" target="_blank" rel="noopener noreferrer">Dodd, Dylan</a>, <a href="http://fischbachgroup.org/" target="_blank" rel="noopener noreferrer">Fischbach, Michael A.</a>, and <a href="http://sonnenburglab.stanford.edu/" target="_blank" rel="noopener noreferrer">Sonnenburg, Justin L.</a> </div> <div class="periodical"> <em>Nature,</em> Jul 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://biorxiv.org/content/10.1101/2021.05.25.445684" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">bioRxiv</a> <a href="https://pubmed.ncbi.nlm.nih.gov/34262212" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PubMed</a> <a href="https://doi.org/10.1038/s41586-021-03707-9" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a> <a href="https://www.nature.com/articles/s41586-021-03707-9.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a> </div> <div class="abstract hidden"> <p>Gut microorganisms modulate host phenotypes and are associated with numerous health effects in humans, ranging from host responses to cancer immunotherapy to metabolic disease and obesity. However, difficulty in accurate and high-throughput functional analysis of human gut microorganisms has hindered efforts to define mechanistic connections between individual microbial strains and host phenotypes. One key way in which the gut microbiome influences host physiology is through the production of small molecules, yet progress in elucidating this chemical interplay has been hindered by limited tools calibrated to detect the products of anaerobic biochemistry in the gut. Here we construct a microbiome-focused, integrated mass-spectrometry pipeline to accelerate the identification of microbiota-dependent metabolites in diverse sample types. We report the metabolic profiles of 178 gut microorganism strains using our library of 833 metabolites. Using this metabolomics resource, we establish deviations in the relationships between phylogeny and metabolism, use machine learning to discover a previously undescribed type of metabolism in <i>Bacteroides</i>, and reveal candidate biochemical pathways using comparative genomics. Microbiota-dependent metabolites can be detected in diverse biological fluids from gnotobiotic and conventionally colonized mice and traced back to the corresponding metabolomic profiles of cultured bacteria. Collectively, our microbiome-focused metabolomics pipeline and interactive metabolomics profile explorer are a powerful tool for characterizing microorganisms and interactions between microorganisms and their host.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Cell Host Microbe</abbr></div> <div id="RN2085" class="col-sm-8"> <div class="title"> <i>Bacteroides thetaiotaomicron</i>-Infecting Bacteriophage Isolates Inform Sequence-Based Host Range Predictions</div> <div class="author"> <a href="https://hrycklab.medicine.wisc.edu/" target="_blank" rel="noopener noreferrer">Hryckowian, Andrew J.</a>,  <em>Merrill, Bryan D.</em>, Porter, Nathan T., Van Treuren, William, Nelson, Eric J., Garlena, Rebecca A., Russell, Daniel A., Martens, Eric C., and <a href="http://sonnenburglab.stanford.edu/" target="_blank" rel="noopener noreferrer">Sonnenburg, Justin L.</a> </div> <div class="periodical"> <em>Cell Host Microbe,</em> Sep 2020 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://biorxiv.org/content/10.1101/2020.03.04.977157" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">bioRxiv</a> <a href="https://pubmed.ncbi.nlm.nih.gov/32652063" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PubMed</a> <a href="https://doi.org/10.1016/j.chom.2020.06.011" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a> <a href="https://www.sciencedirect.com/science/article/pii/S1931312820303528/pdfft?isDTMRedir=true&amp;download=true" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a> </div> <div class="abstract hidden"> <p>Our emerging view of the gut microbiome largely focuses on bacteria, while less is known about other microbial components, such as bacteriophages (phages). Though phages are abundant in the gut, very few phages have been isolated from this ecosystem. Here, we report the genomes of 27 phages from the United States and Bangladesh that infect the prevalent human gut bacterium <i>Bacteroides thetaiotaomicron</i>. These phages are mostly distinct from previously sequenced phages with the exception of two, which are crAss-like phages. We compare these isolates to existing human gut metagenomes, revealing similarities to previously inferred phages and additional unexplored phage diversity. Finally, we use host tropisms of these phages to identify alleles of phage structural genes associated with infectivity. This work provides a detailed view of the gut’s "viral dark matter" and a framework for future efforts to further integrate isolation- and sequencing-focused efforts to understand gut-resident phages.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Science</abbr></div> <div id="RN2201" class="col-sm-8"> <div class="title">Robust variation in infant gut microbiome assembly across a spectrum of lifestyles</div> <div class="author"> <a href="https://mrolm.github.io/" target="_blank" rel="noopener noreferrer">Olm, Matthew R.</a>, Dahan, Dylan, <a href="http://mattcarter.co/" target="_blank" rel="noopener noreferrer">Carter, Matthew M.</a>,  <em>Merrill, Bryan D.</em>, <a href="http://brianyu.org/" target="_blank" rel="noopener noreferrer">Yu, Feiqiao Brian</a>, <a href="https://www.sunitjain.com/" target="_blank" rel="noopener noreferrer">Jain, Sunit</a>, Meng, Xiandong, Tripathi, Surya, <a href="https://hannahwastyk.co/" target="_blank" rel="noopener noreferrer">Wastyk, Hannah C.</a>, Neff, Norma F., Holmes, Susan, <a href="http://sonnenburglab.stanford.edu/" target="_blank" rel="noopener noreferrer">Sonnenburg, Erica D.</a>, <a href="https://nyuad.nyu.edu/en/academics/divisions/science/faculty/aashish-jha.html" target="_blank" rel="noopener noreferrer">Jha, Aashish R.</a>, and <a href="http://sonnenburglab.stanford.edu/" target="_blank" rel="noopener noreferrer">Sonnenburg, Justin L.</a> </div> <div class="periodical"> <em>Science,</em> Jun 2022 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://biorxiv.org/content/10.1101/2022.03.30.486467" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">bioRxiv</a> <a href="https://pubmed.ncbi.nlm.nih.gov/35679413" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PubMed</a> <a href="https://doi.org/10.1126/science.abj2972" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a> <a href="https://www.science.org/doi/pdf/10.1126/science.abj2972?download=true" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a> </div> <div class="abstract hidden"> <p>Infant microbiome assembly has been intensely studied in infants from industrialized nations, but little is known about this process in nonindustrialized populations. We deeply sequenced infant stool samples from the Hadza hunter-gatherers of Tanzania and analyzed them in a global meta-analysis. Infant microbiomes develop along lifestyle-associated trajectories, with more than 20% of genomes detected in the Hadza infant gut representing novel species. Industrialized infants-even those who are breastfed-have microbiomes characterized by a paucity of <i>Bifidobacterium infantis</i> and gene cassettes involved in human milk utilization. Strains within lifestyle-associated taxonomic groups are shared between mother-infant dyads, consistent with early life inheritance of lifestyle-shaped microbiomes. The population-specific differences in infant microbiome composition and function underscore the importance of studying microbiomes from people outside of wealthy, industrialized nations.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge">Cell</abbr></div> <div id="RN2140" class="col-sm-8"> <div class="title">Gut-microbiota-targeted diets modulate human immune status</div> <div class="author"> <a href="https://hannahwastyk.co/" target="_blank" rel="noopener noreferrer">Wastyk, Hannah C.</a>, Fragiadakis, Gabriela K., Perelman, Dalia, Dahan, Dylan,  <em>Merrill, Bryan D.</em>, <a href="http://brianyu.org/" target="_blank" rel="noopener noreferrer">Yu, Feiqiao B.</a>, Topf, Madeline, Gonzalez, Carlos G., Van Treuren, William, <a href="https://thehanlab.com/" target="_blank" rel="noopener noreferrer">Han, Shuo</a>, Robinson, Jennifer L., Elias, Joshua E., <a href="http://sonnenburglab.stanford.edu/" target="_blank" rel="noopener noreferrer">Sonnenburg, Erica D.</a>, Gardner, Christopher D., and <a href="http://sonnenburglab.stanford.edu/" target="_blank" rel="noopener noreferrer">Sonnenburg, Justin L.</a> </div> <div class="periodical"> <em>Cell,</em> Aug 2021 </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://biorxiv.org/content/10.1101/2020.09.30.321448" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">bioRxiv</a> <a href="https://pubmed.ncbi.nlm.nih.gov/34256014" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PubMed</a> <a href="https://doi.org/10.1016/j.cell.2021.06.019" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a> <a href="https://www.sciencedirect.com/science/article/pii/S0092867421007546/pdfft?isDTMRedir=true&amp;download=true" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">PDF</a> </div> <div class="abstract hidden"> <p>Diet modulates the gut microbiome, which in turn can impact the immune system. Here, we determined how two microbiota-targeted dietary interventions, plant-based fiber and fermented foods, influence the human microbiome and immune system in healthy adults. Using a 17-week randomized, prospective study (n = 18/arm) combined with -omics measurements of microbiome and host, including extensive immune profiling, we found diet-specific effects. The high-fiber diet increased microbiome-encoded glycan-degrading carbohydrate active enzymes (CAZymes) despite stable microbial community diversity. Although cytokine response score (primary outcome) was unchanged, three distinct immunological trajectories in high-fiber consumers corresponded to baseline microbiota diversity. Alternatively, the high-fermented-food diet steadily increased microbiota diversity and decreased inflammatory markers. The data highlight how coupling dietary interventions to deep and longitudinal immune and microbiome profiling can provide individualized and population-wide insight. Fermented foods may be valuable in countering the decreased microbiome diversity and increased inflammation pervasive in industrialized society.</p> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Bryan Merrill. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="noopener noreferrer">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="noopener noreferrer">GitHub Pages</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script> <script src="/assets/js/zoom.js"></script> <script src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id="></script> <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","");</script> </body> </html>